src/ImageMetadata.jl

```diff
@@ -21,7 +21,9 @@ export
     data,
     properties,
     shareproperties,
-    spatialproperties
+    spatialproperties,
+    # export for < v"1.2"
+    hasproperty
```

Comment:
Since `hasproperty` is considered to belong `Base`, we should not export this symbol, it's only for internal usage.